src/Benchmarks/project.json

```diff
@@ -6,7 +6,6 @@
   },
   "dependencies": {
     "Dapper": "1.50.0-*",
-    "Microsoft.EntityFrameworkCore.Commands": "1.0.0-*",
```

Comment:
You also need " "Microsoft.EntityFrameworkCore.Tools": "1.0.0-*" in dependencies.

https://docs.efproject.net/en/latest/cli/dotnet.html#installation